Services/Setup/Version320.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Wizzy\Search\Services\Setup;
+
+use Magento\Framework\Setup\SchemaSetupInterface;
+
+class Version320
+{
+    private $setupUtils;
+
+    public function __construct(SetupUtils $setupUtils)
+    {
+        $this->setupUtils = $setupUtils;
+    }
+
+    private $defaultConfigs = [
+        "wizzy_catalogue_configuration/catalogue_configuration_attributes/extra_attributes_to_be_synced" => [],
```

Comment:
As discussed it will be a blank string and make sure we run the patch and see the output.